independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/Codestart.java

```diff
@@ -10,6 +11,11 @@
     private final CodestartSpec spec;
     private final Set<String> implementedLanguages;
 
+    static final Comparator<Codestart> PROCESSING_ORDER = Comparator.comparingInt(Codestart::getTypeOrder)
+            .thenComparing(Codestart::getName);
```

Comment:
I don't think the name should matter in the comparison here.

Reply:
@gastaldi it doesn't, it's to make sure we always use the same ordering when running..